k8s/utils/helper.go

```diff
@@ -6,9 +6,17 @@
 
 package utils
 
-import "regexp"
+import (
+	"regexp"
+
+	apimodel "github.com/eclipse-symphony/symphony/api/pkg/apis/v1alpha1/model"
+)
 
 func IsComponentKey(key string) bool {
 	regex := regexp.MustCompile(`^targets\.[^.]+\.[^.]+`)
 	return regex.MatchString(key)
 }
+
+func IsDeploymentFinished(summary apimodel.SummaryResult) bool {
```

Comment:
Should this be a method on apimodel.SummaryResult itself?

Reply:
Yes, I have moved this to be a method on summaryResult.